src/commands/migration_tasks.py

```diff
@@ -604,43 +604,47 @@ class MigrateCompaniesTask(AbstractFundsTask):
     Migration task for companies
     """
 
+    def __init__(self):
+        """
+        Constructor
+        """
+        self.source_updated = None
+        self.backend_updated = None
```

Comment:
is always unix time as default( if there is a default value for it based on alembic migration).

Reply:
True, but the value needs to be defined because it's loaded in prepare method, not constructor